Ticketing/Ticketing.BAL/Services/RabbitMqService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using log4net;
+using Microsoft.Extensions.Options;
+using RabbitMQ.Client;
+using System.Text;
+using System.Text.Json;
+using Ticketing.BAL.Contracts;
+using Ticketing.BAL.Options;
+
+namespace Ticketing.BAL.Services
+{
+    public class RabbitMqService : IRabbitMqService
+    {
+        private readonly IOptions<RabbitMqSettings> _options;
+        private readonly ILog _logger;
+        public RabbitMqService(IOptions<RabbitMqSettings> options, ILog logger)
+        {
+            _options = options;
+            _logger = logger;
+        }
+
+        public void SendMessage(object obj)
+        {
+            var message = JsonSerializer.Serialize(obj);
+            SendMessage(message);
+        }
+
+        public void SendMessage(string message)
+        {
+            _logger.Debug($"RabbitMqService Send Message to Queue {_options.Value.QueueName}");
+
+            var factory = new ConnectionFactory { HostName = _options.Value.HostName };
```

Comment:
is it possible to be tested?

Reply:
I don't think it is a unit test scenario. 
It is work with external component.